custom_scripts/py_scripts/test_executor/example.py

```diff
@@ -0,0 +1,8 @@
+from typing import Dict, Any
+
+import requests
+
+
+def main(config: Dict[str, Any]):
+    response = requests.head(config['address'])
```

Comment:
It is better to use `.get("address")`, because in this case, you will get `None` if key or value doesn't exist. If you will use it like you use it now, you will get `KeyError` exception (for example, when `config` dictionary doesn't have this key).